packages/contracts/src/contracts/current/protocol/Exchange/MixinSignatureValidator.sol

```diff
@@ -49,6 +56,19 @@ contract MixinSignatureValidator is
         preSigned[hash][signer] = true;
     }
 
+    /// @dev Approves a Validator contract to verify signatures on signer's behalf.
+    /// @param validator Address of Validator contract.
+    /// @param approval Approval or disapproval of  Validator contract.
+    function approveSignatureValidator(
```

Comment:
Should there be a way to "unapprove" a validator?

Reply:
Naming isn't super clear here, but you can pass in `false` to disapprove a validator.